<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Yo-kai Editor B2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="title_label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string notr="true">Yo-kai Editor B2</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>VERSION</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="version_label">
          <property name="text">
           <string notr="true">null</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionOpenDecrypted"/>
    <addaction name="actionSaveDecrypted"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>HELP</string>
    </property>
    <addaction name="actionVersion_information"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>EDIT</string>
    </property>
    <addaction name="actionManual_Edit"/>
    <addaction name="actionCheatCode_Edit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>OPEN...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SAVE AS...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionVersion_information">
   <property name="text">
    <string>ABOUT_THIS_APP</string>
   </property>
  </action>
  <action name="actionManual_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>EDIT_WITH_HEX_EDITOR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionOpenDecrypted">
   <property name="text">
    <string>OPEN_DECRYPTED...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSaveDecrypted">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SAVE_DECRYPTED_AS...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCheatCode_Edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>APPLY_CHEATCODE</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
